Enterprise-CMCS/macpro-mdct-mcr
# [MLR] Re-order MLR Entry metadata and table headers in all instances

### Description



### Related ticket(s)


MDCT-2566

NOTE: this PR also includes updates to the readme for an issue + workaround for DynamoDB issues during setup

---
### How to test

- Login
- Add/create + enter MLR submission
- Navigate to "MLR Reporting" tab on left
- Click "Add program reporting information" button
- Fill + save reporting info
- Confirm the following changes:
    - ![image](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/6752256/026ab1a6-852f-4318-80b0-e472739ddaae)
- Click "Enter MLR" button 
- Confirm the following changes:
    - ![image](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/6752256/11a38ff0-ac90-456d-a2be-80473c2d296b)
- Scroll to bottom, click "Save and Return" button
- Navigate to "Review and Submit" tab on left
- Click "Review PDF" button
- Confirm that the same UI changes are in the PDF

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
